src/tests/homework.enrollment.page.js

```diff
@@ -0,0 +1,77 @@
+import { expect } from "@playwright/test";
+import { faker } from "@faker-js/faker";
+
+export class EnrollmentPage {
+
+    constructor(page) {
+      this.page = page;
+
+      // Locators //
+      this.heading = page.getByRole("heading", { level: 1 });
+      this.fullnameInput = page.getByLabel("Jméno a příjmení");
+      this.emailInput = page.getByLabel("Email");
+      this.passwordInput = page.getByLabel("Heslo");
+      this.confirmPasswordInput = page.getByLabel("Kontrola hesla");
+      this.submitButton = page.getByRole("button", { name: "Zaregistrovat" });
+      this.loggedUser = page.locator(".nav-item").filter({ hasText: "Přihlášen" }).locator("strong");
+      this.alertMessage = page.getByRole("alert");
+    }
+
+    // Actions //
+    async navigate() {
+      await this.page.goto("/registrace");
+    }
+
+    async generateRandomUser() {
```

Comment:
Hehe. Tady sis trošku naběhla, Terez. Pokud test spustíš několikrát za sebou, tak občas ti failne. Dovedla bys investigovat proč? Můžeš to brát jako takovou challenge. zkus si dát na řádek 23:       console.log(randomName, randomEmail, randomPassword);
a na test("New user should register") si dej test.only. Pak pust testy parallelne na 4 workrech a dej repeat:
npx playwright test --workers=4 --repeat-each=5

Reply:
Nepovolené znaky v e-mailu, zdá se, že `faker.internet.email({ allowSpecialCharacters: false })`, které má být default, nefunguje ani nastavené explicitně 😕